wee_slack.py

```diff
@@ -3693,6 +3727,33 @@ def command_status(data, current_buffer, args):
         s = SlackRequest(team.token, "users.profile.set", {"profile": profile}, team_hash=team.team_hash)
         EVENTROUTER.receive(s)
 
+@utf8_decode
+def line_event_cb(data, signal, hashtable):
+    line_pointer = hashtable["_chat_line"]
+    buffer_pointer = hashtable["_buffer"]
+    line_timestamp = hashtable["_chat_line_date"]
+    line_time_id = hashtable["_chat_line_date_printed"]
+
+    e = EVENTROUTER
+    if line_timestamp and line_time_id and buffer_pointer in e.weechat_controller.buffers:
+        ts = SlackTS("{}.{}".format(line_timestamp, line_time_id))
+
+        channel = e.weechat_controller.buffers[buffer_pointer]
+        message_hash = channel.hash_message(ts)
+        if message_hash is None:
+            return w.WEECHAT_RC_OK
+        else:
```

Comment:
No point in having an else-block when the if-block returns.

Reply:
✓